test/unit/state/get-droppable-over/preferencing.spec.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import { type Position, type Rect, getRect } from 'css-box-model';
+import {
+  disableDroppable,
+  getDroppableDimension,
+  getPreset,
+  getDraggableDimension,
+} from '../../../util/dimension';
+import { subtract, patch } from '../../../../src/state/position';
+import { offsetRectByPosition } from '../../../../src/state/rect';
+import type {
+  TypeId,
+  DraggableId,
+  DraggableDimension,
+  DroppableDimension,
+  DroppableDimensionMap,
+  DroppableId,
+} from '../../../../src/types';
+
+import getDroppableOver from '../../../../src/state/get-droppable-over';
+import {
+  getOffsetForEndEdge,
+  getOffsetForCrossAxisEndEdge,
+  getOffsetForCrossAxisStartEdge,
+} from '../get-drag-impact/util/get-offset-for-edge';
+import { toDroppableMap } from '../../../../src/state/dimension-structures';
+import { beforeCrossAxisPoint } from '../../../util/before-point';
+import { afterCrossAxisPoint } from '../../../util/after-point';
+
+const droppableLarge: DroppableDimension = getDroppableDimension({
+  descriptor: {
+    id: 'large',
+    type: 'standard',
+    mode: 'standard',
+  },
+  borderBox: {
+    top: 0,
+    left: 0,
+    right: 600,
+    bottom: 600,
+  },
+});
+
+const droppableSmall: DroppableDimension = getDroppableDimension({
+  descriptor: {
+    id: 'small',
+    type: 'standard',
+    mode: 'standard',
+  },
+  borderBox: {
+    top: 1000,
+    left: 1000,
+    right: 1100,
+    bottom: 1100,
+  },
+});
+
+const droppableSmallSecondary: DroppableDimension = getDroppableDimension({
+  descriptor: {
+    id: 'secondary',
+    type: 'standard',
+    mode: 'standard',
+  },
+  borderBox: {
+    top: 1000,
+    left: 1200,
+    right: 1300,
+    bottom: 1100,
+  },
+});
+
+const draggable: DraggableDimension = getDraggableDimension({
+  descriptor: {
+    id: 'my draggable',
+    index: 0,
+    type: droppableLarge.descriptor.type,
+    droppableId: droppableLarge.descriptor.id,
+  },
+  borderBox: droppableLarge.client.borderBox,
+});
+
+it('should prefer lists that are further away in the case that multiple lists are hit', () => {
```

Comment:
@danieldelcore can you please add a comment explaining which two cases are being hit?